packages/react-internal/src/components/Dialog/__snapshots__/Dialog.deprecated.test.tsx.snap

```diff
@@ -91,7 +91,6 @@ exports[`Dialog deprecated props renders Dialog with className 1`] = `
               transition: opacity 0.267s;
               width: 100%;
             }
-        role="document"
```

Comment:
Might be missing a snapshot update? I wouldn't expect this change now that the Modal changes have been reverted.

Reply:
Updated!